db/migrate/20221208140036_create_bulk_import_rows.rb

```diff
@@ -0,0 +1,10 @@
+class CreateBulkImportRows < ActiveRecord::Migration[6.1]
+  def change
+    create_table :bulk_import_rows do |t|
+      t.references :bulk_import, null: false, foreign_key: { on_delete: :cascade }
+      t.jsonb :data
```

Comment:
I'm starting to wonder whether it makes sense for this to be `jsonb`. `text` should work as well and maybe have lower costs

Reply:
I've somehow missed the usage of `jsonb` in this model. I'm not sure I understand what it's for. For a single CSV row, would the data now be an array of strings?

Reply:
We parse ahead of time and process rows one by one. It's basically the arguments to the worker.

The PostgreSQL documentation seems to recommend using `jsonb` anyway unless we require that the textual representation of the object should be preserved, which is not our case.